data/templates/esx-ks

```diff
@@ -245,7 +245,8 @@ wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%
 esxcli system shutdown reboot -d 10 -r "Rebooting after first boot host configuration"
```

Comment:
@nucklehead I think your branch didn't rebase, so didn't include this commit https://github.com/RackHD/on-http/commit/7340e79b81633a76f335ac53d2def664a984e911, just rebase and submit again.